blocks/carousel/carousel.css

```diff
@@ -193,6 +208,20 @@ main .section div.carousel-wrapper {
 }
 
 /* Media queries */
+
+/* Tablet */
+@media screen and (min-width: 768px) {
+    .carousel.image-carousel-full-width .carousel-slide-container {
+        gap: 61px;
```

Comment:
Where is this `61px` gap coming from? I want to make sure we're using gap spacing that's already in the design systems. Can you touch base/work with @sachinmesh on this? Ideally, we should be using value from the existing gaps that align with our grid.

https://github.com/hlxsites/merative2/blob/main/styles/styles.css#L206